README.md

```diff
@@ -1,25 +1,54 @@
-# Getting Started with Create React App
+# Book Store
+
+Microverse first react/redux project
```

Comment:
- [OPTIONAL] Please it will be nice to include a short description about your project as this will enable users to quickly get an idea of what your project is about.

Reply:
Thanks for your suggestion. I will implement it in the next iteration since it's the feature that it's going to be usable. Happy coding!